stock-tracker/src/store/rootReducer.ts

```diff
@@ -1,6 +1,6 @@
-import { combineReducers } from "redux";
+import { combineReducers, AnyAction } from "redux";
 import { chartReducer } from "../features/chart";
-import { keyStatsReducer } from "../features/keyStats";
+import { keyStatsReducer } from "../features/keyStats/redux/keyStatsReducer";
```

Comment:
Is there any way you can import this just from features/keyStats?

Reply:
It works, but I don't really understand why.  So I don't have to specify file location if I specify enclosing folder?

Reply:
Not exactly - You can do that because in keyStats you have an `index.ts` file. When you import something from a path, if that path is a file it just imports it directly. If it's a folder, then it tries for /index file.